cognite/powerops/prerun_transformations/transformations.py

```diff
@@ -544,10 +561,14 @@ def apply(self, time_series_data: tuple[pd.Series]) -> pd.Series:
             single_ts = time_series_data[0]
             return self.height_to_volume(single_ts, self.heights, self.volumes)
         else:
-            raise ValueError("pre_apply function has not run - missing neccessary properties to run transformation")
+            raise ValueError("pre_apply function has not run - missing necessary properties to run transformation")
 
 
 class DoNothing(Transformation):
```

Comment:
Not really related to this PR, but do we need this one (and if so, what do we use it for)?